src/sidebar/services/frame-sync.js

```diff
@@ -236,8 +236,13 @@ export class FrameSyncService {
     };
 
     const discovery = new Discovery(window, { server: true });
-    discovery.startDiscovery((source, origin, token) =>
-      this._bridge.createChannel({ source, origin, token })
+    discovery.startDiscovery(
+      (source, origin, token) =>
+        this._bridge.createChannel({ source, origin, token }),
+      [
+        // Ping the host frame which is in most cases also the only guest.
```

Comment:
Can you help me understand why it's relevant here that the host frame is usually the only guest? Is it assured that the parent frame is _definitely_ the host?